app/models/main.py

```diff
@@ -1371,6 +1371,40 @@ class query_class(BaseQuery):
         def has_statuses(self, *statuses):
             return self.filter(Brief.status.in_(statuses))
 
+        def has_date_field_after(self, attr, start_datetime, inclusive=None):
```

Comment:
Could we name these methods `datetime_field` so it's really clear by looking at the method name what they take?

Reply:
We can! Done.